ambry-replication/src/main/java/com/github/ambry/replication/continuous/DataNodeTracker.java

```diff
@@ -41,9 +44,11 @@ public class DataNodeTracker {
    * @param remoteReplicas remote replicas for this data node
    * @param maxActiveGroupSize maximum count of replicas in active groups
    * @param startGroupId group id from which we can start and increment and generate unique group id for each group
+   * @param time Ambry time
+   * @param replicaThrottleDurationMs throttle duration for replicas
    */
   public DataNodeTracker(DataNodeId dataNodeId, List<RemoteReplicaInfo> remoteReplicas, int maxActiveGroupSize,
-      int startGroupId) {
+      int startGroupId, Time time, long replicaThrottleDurationMs) {
```

Comment:
Nit: is this the start time for this node or the entire group?

Reply:
This is a time class in ambry that is wrapping java time class. This is used everywhere in ambry. This is used so we can move time forward in our tests.